src/qibolab/platforms/platform.py

```diff
@@ -56,6 +56,18 @@ def set_lo_readout_frequency(self, qubit, freq):
     def get_lo_readout_frequency(self, qubit):
         return self.qubits[qubit].readout.local_oscillator.frequency
 
+    def set_lo_twpa_frequency(self, qubit, freq):
+        self.qubits[qubit].twpa.local_oscillator.frequency = freq
```

Comment:
A way to propagate the error here would be
```suggestion
        if self.qubits[qubit].twpa is None:
            raise_error(NotImplementedError, f"TWPA not available for {self.qubits[qubit].name}")
        self.qubits[qubit].twpa.local_oscillator.frequency = freq
```
but even if you don't do that you will probably get a different error like
```
None type has no attribute local_oscillator
```